CMakeLists.txt

```diff
@@ -666,15 +658,16 @@ endif()
 # End setting options for Profile build
 # #
 
-# When the path starts with a / on a Unix system it's an absolute path.
-# This means that on Windows the path used is always relative.
-if(LOCALEDIR MATCHES "^/")
-	add_definitions(-DHAS_RELATIVE_LOCALEDIR=0)
-	set(LOCALE_INSTALL ${LOCALEDIR})
-else(LOCALEDIR MATCHES "^/")
+# On Windows the path used is always relative.
+if(WIN32)
 	add_definitions(-DHAS_RELATIVE_LOCALEDIR=1)
-	set(LOCALE_INSTALL ${DATADIR}/${LOCALEDIR})
-endif(LOCALEDIR MATCHES "^/")
+	file(RELATIVE_PATH LOCALEDIR "${CMAKE_INSTALL_FULL_BINDIR}" "${CMAKE_INSTALL_FULL_LOCALEDIR}")
+else(WIN32)
+	add_definitions(-DHAS_RELATIVE_LOCALEDIR=0)
+	set(LOCALEDIR "${CMAKE_INSTALL_FULL_LOCALEDIR}")
+endif(WIN32)
+
+add_definitions(-DLOCALEDIR=\\\"${LOCALEDIR}\\\")
```

Comment:
CMP005 has been eliminated. Remove the escapes (\) on both ends they are no longer required.

This is causing a Travis faliure.